content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-#
+
```

Comment:
Rubocop didn't like this

Reply:
For consistency with other methods I would just add a new-line instead of dropping the comment line:

```ruby
# frozen_string_literal: true

#
# Description: calculate entity used quota values
#
```